CMP1903_A1_2324/Program.cs

```diff
@@ -4,16 +4,21 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace CMP1903_A1_2324
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
-        }
+namespace CMP1903_A1_2324 {
+  internal class Program {
+    static void Main(string[] args) {
+      Testing testing = new Testing();
+
+      testing.Test();
+
+      Game game = new Game();
+
+      List<int> tempList = game.RollDie();
+
+      for (int i = 0; i < tempList.Count; i++){
+        Console.WriteLine("Roll " + (i+1) + " is " + tempList[i]);
+      }
+      Console.WriteLine("Sum of the rolls is: " + tempList.Sum());
     }
```

Comment:
All requirements are met for the program class 

Formatting is following the guide to the T

No commenting however, even though code is readable without

No errors occuring